docs/spring-boot/core-starter.md

```diff
@@ -291,6 +291,49 @@ public class SlashHello {
 By default, the `@Slash` annotation is configured to require bot mention in order to trigger the command. You can override
 this value using `@Slash#mentionBot` annotation parameter. 
 
+You can also use slash commands with arguments. To do so, the field `value` of the `@Slash` annotation must have a valid
+format as explained in the [Activity API section](../activity-api.md#Slash-command-pattern-format). 
+If the slash command pattern is valid, you will have to specify all slash arguments as method parameter with the same name and type.
+If slash command pattern or method signature is incorrect, a `warn` message will appear in your application log and
```

Comment:
or would it be better to fail the startup? at least the user is aware he has made a mistake

Reply:
mm the warning was there before... would it be a breaking change to change that....

Reply:
Yes, I would say it is a breaking change. Let's discuss that with @symphony-thibault

Reply:
@symphony-elias @symphony-youri I'd keep the warning for now. We could maybe propose that as a possible improvement and let the community decide (through a GH issue)